openbas-api/src/main/java/io/openbas/service/InjectService.java

```diff
@@ -93,4 +101,50 @@ public void cleanInjectsDocScenario(String scenarioId, String documentId) {
             .toList();
     injectDocumentRepository.deleteAll(updatedInjects);
   }
+
+  @Transactional
+  public InjectResultOverviewOutput duplicate(String id) {
+    Inject injectOrigin = injectRepository.findById(id).orElseThrow(ElementNotFoundException::new);
+    Inject duplicatedInject = InjectUtils.duplicateInject(injectOrigin);
+    duplicatedInject.setTitle(duplicateString(duplicatedInject.getTitle()));
+    Inject savedInject = injectRepository.save(duplicatedInject);
+    return injectMapper.toInjectResultOverviewOutput(savedInject);
+  }
+
+  @Transactional
+  public InjectResultOverviewOutput launch(String id) {
+    Inject inject = injectRepository.findById(id).orElseThrow(ElementNotFoundException::new);
+    inject.clean();
+    inject.setUpdatedAt(Instant.now());
+
+    InjectStatus injectStatus = setInjectStatusAsQueuing(inject);
+    inject.setStatus(injectStatus);
+
+    return injectMapper.toInjectResultOverviewOutput(injectRepository.save(inject));
+  }
+
+  @Transactional
+  public InjectResultOverviewOutput relaunch(String id) {
+    Inject injectOrigin = injectRepository.findById(id).orElseThrow(ElementNotFoundException::new);
+    Inject duplicatedInject = InjectUtils.duplicateInject(injectOrigin);
+
+    Inject savedInject = injectRepository.save(duplicatedInject);
+
+    InjectStatus injectStatus = setInjectStatusAsQueuing(savedInject);
+    savedInject.setStatus(injectStatus);
+
+    injectDocumentRepository.deleteDocumentsFromInject(id);
+    injectRepository.deleteById(id);
+
+    return injectMapper.toInjectResultOverviewOutput(injectRepository.save(savedInject));
```

Comment:
issue: we should avoid duplicating too much code, perhaps we could encapsulate some behaviour